api/mapping/mapmodule/event/GFIResultEvent.md

```diff
@@ -0,0 +1,94 @@
+# GFIResultEvent [rpc]
+
+Notifies user that map gets GFI results.
+
+## Description
+
+Event is used to nofify user about the GFI response. Event includes information about the click coordinates and content of GFI.
```

Comment:
Maybe add something like: "Note! This event is triggered for each layer and vector feature that is hit. You can detect if the events are from the same click by comparing coordinates"?